terraform/aws/modules/mattermost-apps-cloud-deployer/mattermost_apps_cloud_deployer_user_permissions.tf

```diff
@@ -0,0 +1,130 @@
+resource "aws_iam_policy" "mattermost_apps_cloud_deployer_user_policy" {
+  name        = "mattermost-apps-cloud-deployer-policy"
+  path        = "/"
+  description = "IAM user permissions for Mattermost apps cloud deployer user"
+
+  policy = <<EOF
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:ListAllMyBuckets"
+            ],
+            "Resource": "arn:aws:s3:::*"
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:ListBucket",
+                "s3:GetBucketLocation"
+            ],
+            "Resource": [
+                "arn:aws:s3:::${var.static_bucket}",
+                "arn:aws:s3:::${var.terraform_state_bucket}",
+                "arn:aws:s3:::terraform-cloud-monitoring-state-bucket-test"
```

Comment:
@stylianosrigas shouldn't this also be variable?

Reply:
You are right, thanks a lot. Fixed ;)